cmake/linux/CMakeLists.txt

```diff
@@ -2,3 +2,9 @@ INSTALL(FILES lmms.png                  DESTINATION "${DATA_DIR}/pixmaps")
 INSTALL(FILES lmms                      DESTINATION "${DATA_DIR}/menu")
 INSTALL(FILES lmms.desktop              DESTINATION "${DATA_DIR}/applications")
 INSTALL(FILES lmms.xml                  DESTINATION "${DATA_DIR}/mime/packages")
+
+SET(LINUX_DESKTOP_FILE_EXEC "env QT_X11_NO_NATIVE_MENUBAR=1 lmms %f")
+SET(MENU_FILE_ICON_XPM "/usr/share/pixmaps/lmms.xpm")
+SET(LINUX_RELATIVE_DESKTOP_FILE_ICON "lmms")
```

Comment:
Actually on second thought... remove this entirely and use `${CMAKE_PROJECT_NAME}` in the desktop file as this is only used once now that the Debian stuff has been removed. :+1: